pegasus/sites.v3/code.org/views/minecraft_2022_promo_older.haml

```diff
@@ -0,0 +1,36 @@
+:ruby
+  timecraft_image = "/images/mc/fit-720x620/mc_mee_half_timecraft.png"
+  timecraft_title = I18n.t(:hoc2021_mc_mee_title)
+  timecraft_description = I18n.t(:hoc2022_mc_mee_timecraft_description)
+  timecraft_link = "https://aka.ms/HourofCode2021"
+
+  villages_image = "/images/mc/fit-720x620/mc_mee_half_villages.jpg"
```

Comment:
The file in our repo has a `.png` extension. But it looks like maybe it doesn't matter based on your screenshots? Not sure how that works 🤯

Reply:
Whoaaa, how bizarre! This is legacy code too, but I'll update it to use `.png`